### PR TITLE
PRO Dashboard

### PR DESCRIPTION
**Pull request by siandreev**

### CODE REVIEW

File: packages/core/src/service/telegram-oauth.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+export function getWindow(): Window | undefined {
```
Comment (KuznetsovNikita) [inline packages/core/src/service/telegram-oauth.ts]: Could we please keep naming convention for a file names?
all files have name in CamelCase Style

File: packages/uikit/src/hooks/appContext.ts [hunk 2]
```diff
@@ -11,6 +11,8 @@ import {
 } from '@tonkeeper/core/dist/tonkeeperApi/tonendpoint';
 import { Configuration as TronConfiguration } from '@tonkeeper/core/dist/tronApi';
 import React, { useContext } from 'react';
+import webpack from 'webpack';
+import Record = webpack.compilation.Record;
```
Comment (KuznetsovNikita) [inline packages/uikit/src/hooks/appContext.ts]: This lines have an issue during build
Are they matter?

File: packages/core/src/service/telegram-oauth.ts (hunk 1 above)
Comment (siandreev) [reply packages/core/src/service/telegram-oauth.ts]: Done

File: packages/uikit/src/components/dashboard/DashboardTable.tsx [hunk 3]
```diff
@@ -100,8 +101,10 @@ const isNumericColumn = (columnType: DashboardColumnType): boolean => {
 export const DashboardTable: FC<{ className?: string }> = ({ className }) => {
     const { data: columns } = useDashboardColumnsAsForm();
     const { data: dashboardData } = useDashboardData();
-    const { account } = useAppContext();
-    const publicKeys = account?.publicKeys;
+    const { data: wallets, isFetched: isWalletsFetched } = useWalletsState();
+    const mainnetPubkeys = wallets
+        ?.filter(w => w?.network === Network.MAINNET)
```
Comment (KuznetsovNikita) [inline packages/uikit/src/components/dashboard/DashboardTable.tsx]: if `w.network === undefined` it's also mainnet

to get list of all mainnet wallet, please use `w.network !== Network.TESTNET`

File: packages/uikit/src/state/dashboard/useDashboardData.ts [hunk 4]
```diff
@@ -5,33 +5,34 @@ import { useAppContext } from '../../hooks/appContext';
...
         i18n: { language },
         t
     } = useTranslation();
     const selectedColIds = selectedColumns?.map(c => c.id);
-    const publicKeys = account?.publicKeys;
-    const { storage } = useAppSdk();
     const client = useQueryClient();
 
+    const { data: walletsState } = useWalletsState();
+    const mainnetWallets = walletsState?.filter(w => w?.network === Network.MAINNET);
```
Comment (KuznetsovNikita) [inline packages/uikit/src/state/dashboard/useDashboardData.ts]: Same as above

**Pull request merged**